resources/versioning.md

```diff
@@ -0,0 +1,92 @@
+# Versioning policy
+
+This page describes the versioning rules Meilisearch SDKs follow after the release of Meilisearch engine v1.0.0. It explains how and when we should increase the MAJOR, MINOR, and PATCH of the versions.
+
+## 🤖 Basic rules
+
+Meilisearch SDKs releases follow the [SemVer rules](https://semver.org/), including the following basic ones:
+
+> 🔥 Given a version number MAJOR.MINOR.PATCH, increment the:
+>
+> 1. MAJOR version when you make incompatible API changes
+> 2. MINOR version when you add functionality in a backwards compatible manner
+> 3. PATCH version when you make backwards compatible bug fixes
+
+
+**Changes that MAY lead the Meilisearch users (developers) to change their code are considered API incompatibility and will make us increase the MAJOR version of Meilisearch SDK.**
+
+**In other terms, if the users MAY have to do more steps than just downloading the new Meilisearch package and running it, a new MAJOR is needed.**
+
+Examples of changes making the code break and then, involving increasing the MAJOR:
+
+- Any change related to the public API the SDK exposes like:
+    - Renaming a method name or classes
+    - Removing or changing a positional argument of a method.
+
+## Reasons why we should release a major version
+
+### New runtimes adoption
+
+We don't have a strict policy yet regarding the minimum support of the language version. Example:
+
+Ruby SDK has a `required_ruby_version >= 2.6`, which prevents users from using the gem if they run their apps with a `Ruby 2.5`.
+
+It is often good to only allow newer versions of the runtimes because they are maintained :sparkles: and there are always new features we can adopt in the code.
+
+But there is a price for that. When we make that cut, we reduce the number of users who can adopt our SDK/service.
+
+Currently, all supported versions are set in the CI of each repository.
+
+This is the list of minimum supported language version runtimes*:
+
+| Language/Framework | Version |
+| ------------------ | ------- |
+| .NET | netstandard2.0 |
+| Dart | >= 2.12.0 |
+| Go | >= 1.16 |
+| Java | >= 8.0 |
+| JavaScript/Node | >= 14 |
+| Swift | >= 5.2 |
+| PHP | >= 7.4 |
+| Python | >= 3.7 |
+| Ruby | >= 2.6 |
+| Rust | N/A |
```

Comment:
Does that mean we will have to update it each time we drop the support of a version?

Reply:
Yeah! But this kind of change does not happen in a high-frequency, may once a year or more?